### ISSUE TITLE
PageRangeDisplayed=2 not working when first page is selected

### ISSUE THREAD

**Issue by miestr** [AdeleD/react-paginate#307]
My setup: total 12 pages
```
 pageRangeDisplayed={2}
marginPagesDisplayed={1}
```
This means that margin pages 1 and 12 (1 per margin) are always displayed, and when I am on page 7, I get neighbour pages 6 and 8 (2 buttons in range, 1 per side) offered - this is correct.
![valid](https://user-images.githubusercontent.com/22884148/81297002-fdfb5580-9072-11ea-8410-80544dd109c5.PNG)
But when I am on page 1, I expect pages 1, 2 and 12 to be offered.
![expectedResult](https://user-images.githubusercontent.com/22884148/81297020-0489cd00-9073-11ea-85a1-754613a08b52.PNG)

 However, I get pages 1, 2, 3 and 12. Why? Page number 3 shal be out out of my range. Neighbours for page 1 are buttons 0 and 2, but as pase 0 does not exist, I shall only get the page on the right (so page number 2).
![invalidResult](https://user-images.githubusercontent.com/22884148/81297049-0a7fae00-9073-11ea-8bfd-b9a54a069584.PNG)

Basically, this is confusing from UX perspective, because if I am on page number 1, I want to navigate either to the page next to it, or may be to the last one. But not the page number 3.
How to achieve such a behaviour?

**Comment by AXV1140** [comment]
any solution for this

**Comment by MonsieurV** [comment]
This should be fixed by #363.

Could someone validate this works for them?

To install from GitHub `master`:

```sh
npm install --save github:AdeleD/react-paginate#master
```

Thanks for the reports!

@AdeleD If you see this message, I think we could release the `v7.2.0` (I didn't tag atm)

**Comment by AdeleD** [comment]
FYI I've just published v8.0.0
